docs/uptime/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+[role="xpack"]
+[[xpack-uptime]]
+= Uptime
+
+Use the Uptime UI to monitor the status of network endpoints via HTTP/S, TCP,
+and ICMP. You will be able to explore status over time, drill into specific monitors,
+and view a high-level snapshot of your environment at a selected point in time.
+
+== Add monitors
```

Comment:
Cleaned up the grammar a bit and added a link to the Heartbeat ref. Is this the right place to link?
```
To get started with Uptime monitoring, you'll need to define monitors.
Monitors are defined in Heartbeat's configuration file.
See {heartbeat-ref}/heartbeat-configuration.html[Configure Heartbeat] for instructions
on configuring monitors to begin storing Uptime information in your cluster.
```